isomorphe/app.py

```diff
@@ -174,6 +175,26 @@ def transform_original(job_id: str, uuid: str):
     return Response(result.original, mimetype="text/xml", headers={"Content-Type": "text/xml"})
 
 
+@app.route("/transform/success/<job_id>/diff/<uuid>")
+def transform_diff(job_id: str, uuid: str):
+    job = get_job(job_id)
+    if not job or not job.result:
+        abort(404)
+    result: SuccessTransformBatchRecord | None = next(
+        (j for j in job.result.records if j.uuid == uuid), None
+    )
+    if not result or not result.original or not result.result:
+        abort(404)
+    diff = difflib.unified_diff(
+        result.original.decode("utf-8").splitlines(),
+        result.result.decode("utf-8").splitlines(),
+        fromfile="original.xml",
```

Comment:
I'd go with `fromfile` = `tofile` = "XML" or something like that.

![image](https://github.com/user-attachments/assets/ba8fd588-b754-48c9-9166-a8966abbbd4d)

seems more to the point than:

![image](https://github.com/user-attachments/assets/c166687d-552a-4df3-90dc-d3df42f67b0b)

Reply:
UUID, even better!